README.md

```diff
@@ -1,2 +1,57 @@
-# taverna-integrations
-Building integrations for Taverna app
+[![Coverage Status](https://coveralls.io/repos/github/teamtaverna/slack_integration/badge.svg?branch=master)](https://coveralls.io/github/teamtaverna/slack_integration?branch=master) [![Build Status](https://travis-ci.org/teamtaverna/slack_integration.svg?branch=master)](https://travis-ci.org/teamtaverna/slack_integration)
+
+# Slack Integration
+Slackbot for Taverna app, an open source meal review and management platform. More information can be found on the [wiki](https://github.com/teamtaverna/taverna/wiki).
+
+### Tech
+
+Taverna slackbot is written in Python3 and built on [lins05/slackbot](https://github.com/lins05/slackbot) library.
+
+### Collaboration
+
+Want to contribute? Great!
+
+Clone the repository from [GitHub](https://www.github.com)
+```
+$ git clone https://github.com/teamtaverna/slack_integration.git
+```
+
+Taverna slackbot depends on [Taverna Core](https://github.com/teamtaverna/core), its API. You'd need to set up  [Taverna Core](https://github.com/teamtaverna/core) project also in order to work effectively locally.
+
+### Installation
+
+**Mac Users**
```

Comment:
If you say "Mac Users", does that not mean we should have for other platforms as well?